src/fidesops/api/v1/scope_registry.py

```diff
@@ -1,3 +1,6 @@
+from fidesops.api.v1.urn_registry import USER_PASSWORD_RESET
```

Comment:
Is there a particular reason for importing the `USER_PASSWORD_RESET` urn at the top? It looks line it gets reassigned on line 54 to the scope

Reply:
That was my IDE getting up to no good. FIxed.